examples/mysql-wordpress-pd/README.md

```diff
@@ -278,6 +326,10 @@ NAME        CLUSTER-IP     EXTERNAL-IP     PORT(S)   AGE
 wordpress   10.0.0.5       1.2.3.4         80/TCP    19h
 ```
 
+**Note:** if you're using Host Path for storage then the wordpress service's
+`LoadBalancer` type is not implemented, and thus the wordpress pod's external
+IP address is not available (remains "Pending").
+
 ## Visit your new WordPress blog
```

Comment:
@jeffmendoza Thanks for the links. I've pushed my latest changes. I am still not able to get an external ip in my test env but I don't think that matters for the changes I'm submitting. Please allow a merge if this LGTY.  Thanks!